src/test/data/JsonSerializableGuestBookTest/typicalGuestGuestBook.json

```diff
@@ -7,54 +7,61 @@
     "dateRange": "13/09/22 - 15/09/22",
     "numberOfGuests": "1",
     "isRoomClean": "yes",
-    "bill": "0.00"
+    "bill": "0.00",
+    "request" : "I wanna sleep"
   }, {
     "name": "Benson Meier",
     "phone": "98765432",
     "email": "johnd@example.com",
     "dateRange": "30/09/22 - 02/10/22",
     "numberOfGuests": "2",
     "isRoomClean": "yes",
-    "bill": "10.00"
+    "bill": "10.00",
+    "request" : "I wanna sleep"
   }, {
     "name": "Carl Kurz",
     "phone": "95352563",
     "email": "heinz@example.com",
     "dateRange": "30/12/22 - 02/01/23",
     "numberOfGuests": "3",
     "isRoomClean": "no",
-    "bill": "19.99"
+    "bill": "19.99",
+    "request" : "I wanna sleep"
```

Comment:
Should we have more probable request examples since it represents a typical GuestBook?